test/functional/page_objects/common_page.js

```diff
@@ -359,7 +359,11 @@ export function CommonPageProvider({ getService, getPageObjects }) {
     }
 
     async getBodyText() {
-      const el = await find.byCssSelector('body>pre');
+      if (await find.existsByCssSelector('a[id=rawdata-tab]')) {
+        // Firefox has 3 tabs and requires navigation to see Raw output
+        await find.clickByCssSelector('a[id=rawdata-tab]');
+      }
+      const el = await find.byCssSelector('body pre');
```

Comment:
Firefox has quite different DOM on that page, so this way is the only I found to work on both Chrome/Firefox